Lib/fontTools/subset/__init__.py

```diff
@@ -2902,7 +2902,7 @@ def main(args=None):
             text += g[7:]
             continue
         if g.startswith('--text-file='):
-            text += open(g[12:]).read().replace('\n', '')
+            text += open(g[12:], encoding='utf-8').read().replace('\n', '')
```

Comment:
It works with python 2.7.10, I have tried myself.
The open method is not io.open().